kernelci.org/content/en/blog/posts/2024/strategic-updates/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2024-04-08
+title: "Strategic updates from the KernelCI project"
+linkTitle: "Strategic updates from the KernelCI project"
+author: Gustavo Padovan
+description: >
+    Check out our strategic overview of the steps we have taken towards KernelCI mission over the past few months.
+---
+
+The KernelCI project continues on its mission to ensure the quality, stability and long-term maintenance of the Linux kernel. That means supporting the community (especially maintainers) to not just run their code in a Continuous Integration (CI) system, but also deliver relevant, high confidence results and reports. In this post, we will give you a strategic overview of the steps we have taken towards our mission over the past few months.
+
+## Enabling new infrastructure to run tests
+
+Our legacy system has shown its age and it has been failing to meet the growing testing needs of the community. It is a quite limited, unstable python2 project that focuses on embedded hardware. We continue on the journey to put in place our new infrastructure, so we can finally move away from the legacy KernelCI system.
+
+We bring good news! The new core service for running tests is already in place, but still going through a stabilization phase. So, the team is ramping up the number of tests slowly to deal with issues that arise especially when it comes to the quality of the testing KernelCI provides. We do not want to repeat past mistakes with results that can't really be trusted, so our focus right now is on quality rather than quantity. The team is iterating quickly on that process to enable open, wide adoption in the coming months.
+
+In our new KernelCI infrastructure, we already have support to run tests in labs (through [LAVA](https://www.lavasoftware.org/) only at the moment), VM, Docker containers and Kubernetes. Adding new labs or test environments should be relatively straightforward. Then, as we add more test environments, we are also laying down the foundation for integration with other CI systems across the community so we can share kernel builds and offer test environments.
```

Comment:
minor: nitpick: I'd drop the VM part which I believe refers to the `shell` runtime. It doesn't spawn/submit any new job like the other runtimes - it just executes without setting up specific environment (VM is not a requirement).

```suggestion
In our new KernelCI infrastructure, we already have support to run tests in labs (through [LAVA](https://www.lavasoftware.org/) only at the moment), Docker containers, Kubernetes and natively. Adding new labs or test environments should be relatively straightforward. Then, as we add more test environments, we are also laying down the foundation for integration with other CI systems across the community so we can share kernel builds and offer test environments.
```